net-mgmt/telegraf/src/opnsense/mvc/app/models/OPNsense/Telegraf/Input.xml

```diff
@@ -114,5 +114,12 @@
         <unbound type="BooleanField">
             <Required>N</Required>
         </unbound>
+        <apcupsd type="BooleanField">
+            <Required>N</Required>
+        </apcupsd>
+        <apcupsd_server type="TextField">
+            <default>127.0.0.1</default>
```

Comment:
I don't think this validates when enabled and left empty? there is no guard around the apcupsd_server field in the template or Required set to Y

Reply:
You're right.
I could set it to required or set the default to `127.0.0.1` in the config template [here](https://github.com/opnsense/plugins/pull/3177/files#diff-e3501e5c04617b45da71f3f7a10aca5108f398144b954d771b0bc47f8d22c0d9R308)

Which method do you prefer?

Reply:
default to template then, don't have to overcomplicate

But I would change the type to `NetworkField` for better validation

Reply:
I set the default to `127.0.0.1` in the config template and changed the help text.

`NetworkField` only allows IPs.
Some setups may use an hostname for this, so i would leave if as a `TextField`.